indykite/resource_authorization_policy.go

```diff
@@ -60,11 +58,25 @@ func resourceAuthorizationPolicy() *schema.Resource {
 				Required:         true,
 				DiffSuppressFunc: structure.SuppressJsonDiff,
 				ValidateFunc: validation.All(
+					validation.StringIsNotEmpty,
 					validation.StringIsJSON,
-					authorizationPolicyValidateJSON,
 				),
 				Description: "Configuration of Authorization Policy in JSON format, the same one exported by Console.",
 			},
+			statusKey: {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringInSlice(getMapStringKeys(AuthorizationPolicyStatusTypes), true),
```

Comment:
Here you set `true` to ignore case. But if someone will write it lowercased, it will not work. Fix your code (and maybe include in tests too) both cases. Or keep it case sensitive

Reply:
Still case insensitive, but do not handling that in code.

Reply:
If you would like to make it case insensitive, you would have to implement `DiffSuppressFunc` on the schema. But it is pretty tricky to define that function well. So I wouldn't go that way

Reply:
Thank you, I asked just for knowledge